listenbrainz/webserver/views/profile.py

```diff
@@ -258,3 +261,23 @@ def connect_spotify_callback():
         flash.warn('Unable to authenticate with Spotify (error {})'.format(e.args[0]))
 
     return redirect(url_for('profile.connect_spotify'))
+
+
+@profile_bp.route('/refresh-spotify-token', methods=['GET'])
```

Comment:
This feels like it should be POST, not GET

Reply:
It's defined as such for the api:
https://github.com/metabrainz/listenbrainz-server/blob/f61e811ddee8911990dcfdf9893115be0646c816/listenbrainz/webserver/views/api.py#L294